src/instructlab/model/train.py

```diff
@@ -692,9 +731,16 @@ def _prepare_phased_base_dir(phased_base_dir: pathlib.Path) -> None:
     phase2_dir_path = phased_base_dir / "phase2"
 
     for p in [phase1_dir_path, phase2_dir_path]:
-        utils.clear_directory(p)
+        if delete_subdirs:
+            utils.clear_directory(p)
         _setup_phase_dirs(p)
 
+    journalfile_path = phased_base_dir / "journalfile.yaml"
```

Comment:
The journalfile logic file handling logic shouldn't exist here IMO. We should move this to the Journalfile class instead.

Reply:
moved file creation into class. kept file deletion in phased dir setup function since destruction already happens there.